scripts/postgres_run.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Thing shim around the normal Docker postgres entrypoint that allows us to run
+# non-application migrations. Things like DB and user creations that would be
+# done by cloud-automation tasks in a normal env.
+
+set -e
+
+# Initialize the DB, but don't allow outside connections yet.
+docker-entrypoint.sh postgres -c listen_addresses='127.0.0.1' &
+# Wait until the server is out of initialization mode and online.
+while ! psql -U postgres -h localhost -c 'SELECT 1;' 2>/dev/null; do sleep 1; done
```

Comment:
i would add a log so that we can pinpoint what's going on when we wait / if we never exit the loop

Reply:
Great point. Thanks! ➿